src/styles/variables.scss

```diff
@@ -0,0 +1,6 @@
+$font: 'Roboto', sans-serif;
+$mainColor: #060b35;
+$secondColor: #616070;
+$fontWeightStandard: 400;
+$active-color: #00acdc;
```

Comment:
The variable name `$active-color` uses a kebab-case naming convention, which is not consistent with the camelCase convention used in other variable names. To maintain consistency, consider renaming it to `$activeColor` following the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).